mrtt-ui/README.md

```diff
@@ -1,46 +1,5 @@
-# Getting Started with Create React App
+# WIP: currently just notes not to forget for a later more refiled readme
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Dev notes:
 
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
```

Comment:
Can we keep some of this as it is still relevant? In the future we can update and modify as required.
I plan to add more specific info to the `gmw-users` root README.md as as part of https://github.com/globalmangrovewatch/gmw-users/issues/40 as well

Reply:
agreed! otherwise plop sounds great hah

Reply:
sure.